README.md

```diff
@@ -12,6 +12,14 @@ There are **module helper** functions to check if a module or theme exists in th
 
 The core module also provides a command and the graphql mutation for **email sending**.
 
+## Installation
+
+Currently suggested approach is to use git submodules. It will be replaced with pOS module manager in the feature. For now, ensure `modules` directory exists in your root directory and add a git submodule:
+
+`mkdir modules && cd modules && git submodule add git@github.com:Platform-OS/pos-module-core.git`
```

Comment:
how about somethink like
```sh
mkdir modules && git submodule add --name core git@github.com:Platform-OS/pos-module-core.git modules/core
```

just to make sure it also has the right module _name_ and _path_

Reply:
yup, thanks!